File: app/src/main/java/io/twere/plainpure/data/api/dribbble/DribbbleTeamsService.java
package io.twere.plainpure.data.api.dribbble;

import io.twere.plainpure.mvp.model.dribbble.Shot;
import io.twere.plainpure.mvp.model.dribbble.User;
import java.util.List;
import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

public interface DribbbleTeamsService {

    @GET("teams/{team}/members") Observable<List<User>> getTeamMembersList(@Path("team") long teamId);
    
    @GET("teams/{team}/shots")
    Observable<List<Shot>> getTeamShotsList(@Path("team") long teamId);
}

File: app/src/main/kotlin/io/twere/plainpure/data/api/DribbbleTeamsService.kt
package io.twere.plainpure.data.api

import io.twere.plainpure.data.local.shot.Shot
import io.twere.plainpure.data.local.user.User
import retrofit2.http.GET
import retrofit2.http.Path
import rx.Observable

interface DribbbleTeamsService {

  @GET("teams/{team}/members")
  fun getTeamMembersList(
      @Path("team") teamId: Long): Observable<List<User>>

  @GET("teams/{team}/shots")
  fun getTeamShotsList(
      @Path("team") teamId: Long): Observable<List<Shot>>

}